client/my-sites/email/gsuite-add-users/add-users.jsx

```diff
@@ -326,7 +323,6 @@ AddEmailAddressesCard.propTypes = {
 	domains: PropTypes.array.isRequired,
 	isRequestingSiteDomains: PropTypes.bool.isRequired,
 	gsuiteUsers: PropTypes.array.isRequired,
-	gsuiteUsersLoaded: PropTypes.bool.isRequired,
```

Comment:
No, I moved [control up to the parent]( https://github.com/Automattic/wp-calypso/pull/31444/files/f41117e28ac1820b5debeb2288a50c7c59cf3947#diff-2978fc814739279fb803847c5c02d7fdR70) so that this component is not rendered we have the G Suite Users loaded already.